ssl/statem/statem_srvr.c

```diff
@@ -1680,6 +1663,30 @@ MSG_PROCESS_RETURN tls_process_client_hello(SSL_CONNECTION *s, PACKET *pkt)
         goto err;
     }
 
+    if (SSL_CONNECTION_IS_DTLS(s)) {
+        int minversion, maxversion;
+
+        /*
+         * If the connection supports DTLSv1.3 or if the ClientHello was sent
+         * with the SupportedVersions extension:
```

Comment:
What is the link between presence of `supported_versions` and tolerating empty cookies? Is it valid for `supported_versions` to include only DTLS 1.2 and prior? It is not immediately obvious how these are connected.